docs/pages/setup/guides/ssh-key-extensions.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Using Teleport CA with Github.
+description:  How to use Teleport SSH short lived certificates with GitHub.
+---
+
+Teleport supports exporting user SSH certificates with configurable key extensions, this allows the Teleport CA to be used in conjunction with GitHubs support for SSH Certificate Authorities to allow users to access their organizations repositories with short lived, signed SSH keys.
```

Comment:
```suggestion
Teleport supports exporting user SSH certificates with configurable key extensions, this allows the Teleport CA to be used in conjunction with GitHub's support for SSH Certificate Authorities to allow users to access their organizations repositories with short lived, signed SSH certificates.
```

Keys or certificates?

Reply:
I think keys makes sense here, they'd be the keys that the user would be using with git/ssh, I'm not sure though

Reply:
In addition to Zac's suggestions:

```suggestion
Teleport supports exporting user SSH certificates with configurable key extensions. This allows the Teleport CA to be used in conjunction with GitHubs support for SSH Certificate Authorities. This way, users can access their organizations' repositories with short-lived, signed SSH keys.
```